tester_app/client/app/bundles/HelloWorld/store/helloWorldStore.jsx

```diff
@@ -0,0 +1,24 @@
+import { compose, createStore, applyMiddleware, combineReducers } from 'redux';
+import thunkMiddleware from 'redux-thunk';
+import loggerMiddleware from 'lib/middlewares/loggerMiddleware';
+import reducers from '../reducers';
+import { initalStates } from '../reducers';
+
+export default props => {
+  const initialName = props.initialName;
+  const { $$helloWorldState } = initalStates;
+  const initialState = {
```

Comment:
@robwise how's this:

// initialState is used by redux store initialization and it expects an Object, rather than an Immutable.Map.